components/form/builder/src/index.js

```diff
@@ -189,15 +188,17 @@ class FormBuilder extends Component {
     return (
       <Select
         errors={errors}
-        showErrors={showErrors}
+        isSubmited={isSubmited}
```

Comment:
I think the correct spelling is `submitted` ;)

Reply:
ups... thx!